### ISSUE TITLE
Subscribed to Newsletter label and checkbox not aligned proper in admin panel while any customer editing 

### ISSUE THREAD

**Issue by cedarvinda** [magento/magento2#19905]
**Subscribed to Newsletter** label and **checkbox** not aligned proper in admin panel while any customer editing 

### Preconditions (*)

1. Magento 2.3.0
2. PHP 7.2

### Steps to reproduce (*)

 Step 1: Go to admin url
 Step 2: login to admin panel
 Step 3: click on **Customers** link from left side and under customers click on **all customers** 
 Step 4: You see all registered customer grid now you **edit** any one of them.
 Step 5: Now click on **newsletter** tab (and take ref from screenshot)

### Expected result (*)

![sol](https://user-images.githubusercontent.com/26018716/50288704-ad3e8900-048c-11e9-8811-34410d1ec3b8.png)

### Actual result (*)

![issues](https://user-images.githubusercontent.com/26018716/50288719-b596c400-048c-11e9-969b-ba798d81559e.png)

**Comment by magento-engcom-team** [comment]
Hi @cedarvinda. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@cedarvinda do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by magento-engcom-team** [comment]
Hi @GovindaSharma. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [x] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [x] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [x] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [x] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by GovindaSharma** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @GovindaSharma. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @GovindaSharma, here is your [Magento instance](https://i-19905-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-19905-2-3-develop.instances.magento-community.engineering/admin](https://i-19905-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
@GovindaSharma Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-97264`, `MAGETWO-97265` were created

**Comment by magento-engcom-team** [comment]
@cedarvinda, thank you for your report.     
    
Unfortunately, we are archiving this ticket now as it did not get much attention from both Magento Community and Core developers for an extended period. This is done in an effort to create a quality, community-driven backlog which will allow us to allocate the required attention more easily.

Please feel free to comment, reopen or create new ticket according to the [Issue reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines) 
if you are still facing this issue on the latest `2.3-develop` branch. Thank you for collaboration.
